FindOnPage.md

```diff
@@ -0,0 +1,619 @@
+        /// To change the ongoing search, StartFind must be called again with a new or modified configuration object.
+        /// This method is primarily designed for HTML document searches.
+        Windows.Foundation.IAsyncAction StartFindAsync(CoreWebView2FindConfiguration configuration);
+
+        /// Navigates to the next match in the document.
+        void FindNext();
+
+        /// Navigates to the previous match in the document.
+        void FindPrevious();
+
+        /// Stops the current 'Find' operation and hides the Find bar.
+        void StopFind();
+
+        /// Gets or sets the state of whether all matches are highlighted. Returns TRUE if all matches are highlighted, FALSE otherwise.
+        Boolean ShouldHighlightAllMatches { get; set; };
+
+        /// Checks if a custom user interface is desired by the end developer. Returns TRUE if using a custom UI, FALSE if using the default.
+        Boolean SuppressDefaultDialog { get; set; };
+
+        /// Retrieves the index of the currently active match in the find session. Returns the index of the currently active match, or -1 if there is no active match.
+        UInt32 ActiveMatchIndex { get; };
+
+        /// Gets the total count of matches found in the current document based on the last search criteria. Returns the total count of matches.
+        UInt32 MatchCount { get; };
+        
+        /// Registers an event handler for the MatchCountChanged event. This event is raised when the total count of matches in the document changes due to a new search operation or changes in the document. The parameter is the event handler to be added. Returns a token representing the added event handler. This token can be used to unregister the event handler.
+        [event_handler("", "", "")]
+        event Windows.Foundation.TypedEventHandler<CoreWebView2Find, Object> FindMatchCountChanged;
+
+        /// Registers an event handler for the ActiveMatchIndexChanged event. This event is raised when the index of the currently active match changes. This can happen when the user navigates to a different match or when the active match is changed programmatically. The parameter is the event handler to be added. Returns a token representing the added event handler. This token can be used to unregister the event handler.
+        [event_handler("", "", "")]
+        event Windows.Foundation.TypedEventHandler<CoreWebView2Find, Object> FindActiveMatchIndexChanged;
+    };
+}
+```
+
+# Appendix
+
+This API specification focuses on providing developers with the necessary information 
+to integrate text finding and navigation functionalities into WebView2 applications. 
+It emphasizes the usage of interfaces such as `ICoreWebView2Find` and 
+`ICoreWebView2FindConfiguration` to perform find operations effectively. 
+
+Additional Info:
+Starting a find session when one is in progress will result in the active match index
```

Comment:
In this situation it will not start at the 2nd or 3rd index etc, it will start from the beginning. When the find query is changed, it starts a completely new find session.